webapp/src/main/webapp/src/app/pages/tests/create/test-create.page.ts

```diff
@@ -13,28 +13,34 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Component, OnInit} from "@angular/core";
-import { ActivatedRoute, Router, ParamMap } from "@angular/router";
+import { Component, OnInit } from "@angular/core";
```

Comment:
These are a lot of formatting changes that make it more difficult to review for merging. Please keep changes related to PR purpose.